extensions/postgresql-metadata-storage/src/main/java/io/druid/metadata/storage/postgresql/PostgreSQLConnector.java

```diff
@@ -51,6 +51,8 @@ public PostgreSQLConnector(Supplier<MetadataStorageConnectorConfig> config, Supp
     datasource.setDriverClassName("org.postgresql.Driver");
 
     this.dbi = new DBI(datasource);
+
+    log.info("Configured PostgreSQL as the metadata storage");
```

Comment:
I believe the point of these logs is to make it so that they are in the logs by default.  It makes it a lot easier when trying to figure out how a system is actually working when debugging other people's stuff.